ubcpi/static/html/ubcpi.html

```diff
@@ -19,27 +19,31 @@ <h2 id="pi-question-h" class="question-text">{{display_name}}</h2>
             </div>
 
             <a name="reflecting"></a>
-            <span id="reflect" ng-if="rc.status() == rc.ALL_STATUS.ANSWERED">
-
-                <ul style="background:#f8f8f8;border:2px solid #ddd;margin:1em 0;padding-top:.3em;text-transform:uppercase;">
-                    <li style="display:inline-block;width:30%;text-align:center;color:#aaa;padding-top:.3em"><span class="fa fa-check-circle" style="font-size:1.4em;"></span></span><br>Answer</li>
-                    <span class="fa fa-angle-double-right" style="color:#ddd;vertical-align:15%;font-size:2.2em"></span>
-                    <li style="display:inline-block;width:30%;text-align:center;font-weight:bold;color:#333;padding-top:.3em;text-transform:uppercase;"><span class="fa fa-arrow-circle-down" style="font-size:1.4em;"></span><br>Reflection</li>
-                    <span class="fa fa-angle-double-right" style="color:#ddd;vertical-align:15%;font-size:2.2em"></span>
-                    <li style="display:inline-block;width:30%;text-align:center;color:#aaa"><span class="fa fa-circle-o" style="font-size:1.4em;"></span><br>Results</li>
-                </ul>
-            </span>
+            <div role="group" aria-label="Progress Indicator">
+                <span id="reflect" ng-if="rc.status() == rc.ALL_STATUS.ANSWERED">
+                    <ol style="background:#f8f8f8;border:2px solid #ddd;margin:1em 0;padding-top:.3em;text-transform:uppercase;">
+                        <li style="display:inline-block;width:30%;text-align:center;color:#414141;padding-top:.3em"><span class="fa fa-check-circle" aria-hidden="true" style="font-size:1.4em;"></span></span><br>Answer</li>
+                            <span class="fa fa-angle-double-right" style="color:#ddd;vertical-align:15%;font-size:2.2em" aria-hidden="true"></span>
+                        <span class="sr">, Completed</span>
```

Comment:
@jleong-openedx the status information needs to be within the ```<li>``` element so that it is read along with the name of the Step in the process "Answer, Completed".